lib/index.js

```diff
@@ -45,18 +41,25 @@ Embark = {
   },
 
   deployContracts: function(env, contractFiles, destFile, chainFile, withProvider, withChain, cb) {
+    var web3 = this.web3;
+    web3.setProvider(new web3.providers.HttpProvider("http://" + this.blockchainConfig.rpcHost + ":" + this.blockchainConfig.rpcPort))
```

Comment:
`this.blockchainConfig.rpcHost` probably should be replaced by `localhost`, especially since this is sending stuff in plaintext via http.